app/src/main/java/com/hammer/talkbbokki/presentation/comment/CommentDetailScreen.kt

```diff
@@ -0,0 +1,244 @@
+package com.hammer.talkbbokki.presentation.comment
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.itemsIndexed
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Icon
+import androidx.compose.material.Text
+import androidx.compose.material.TextField
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import androidx.hilt.navigation.compose.hiltViewModel
+import com.hammer.talkbbokki.R
+import com.hammer.talkbbokki.ui.theme.*
+
+@Composable
+fun CommentDetailRoute(
+    onBackClick: () -> Unit,
+    viewModel: CommentsViewModel = hiltViewModel()
+) {
+    val recomments = viewModel.commentItems.collectAsState()
+    CommentDetailScreen(
+        onBackClick,
+        recomments.value.first(),
+        recomments.value
+    )
+}
+
+@Composable
+fun CommentDetailScreen(
+    onBackClick: () -> Unit,
+    comment: Comment,
+    recomments: List<Comment>
+) {
+    Box(
+        Modifier
+            .fillMaxSize()
+            .background(MainBackgroundColor)
+    ) {
+        Column(
+            Modifier
+                .fillMaxSize()
+                .padding(bottom = 78.dp)
+        ) {
+            CommentDetailHeader(onBackClick)
+            CommentContents(
+                nickname = comment.nickname,
+                date = comment.date,
+                content = comment.content,
+                replyCount = comment.replyCount,
+                onDeleteClick = comment.onDeleteClick
+            )
+            RecommentList(
+                recomments
+            )
+        }
+        Box(Modifier.align(Alignment.BottomCenter)) {
+            RecommentInputArea()
+        }
+    }
+}
+
+@Composable
+fun CommentDetailHeader(
+    onBackClick: () -> Unit,
+) {
+    val replyCount = 0
+    Box(
+        Modifier
+            .height(56.dp)
+            .fillMaxWidth()
+    ) {
+        Icon(painter = painterResource(id = R.drawable.ic_arrow_left),
```

Comment:
🚫 **[ktlint]** <standard:argument-list-wrapping> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Argument should be on a separate line (unless all arguments can fit a single line)